style.css

```diff
@@ -0,0 +1,410 @@
+@media only screen and (min-width: 1080px) {
+  nav {
+    position: sticky;
+    top: 0;
+    background-color: #fff;
+  }
+}
+
+@media (min-width: 1300px) {
+  body {
+    justify-content: center;
+  }
+}
+
+* {
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+body {
+  display: grid;
+  grid-template-rows: repeat(4, auto);
+  margin: 5em;
+  justify-content: center;
+}
+
+#home {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
+  position: sticky;
+  top: 0;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.navbar {
+  width: 1440px;
+}
+
+.logo-section {
+  padding: 10px 12px;
+  margin-left: 4%;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.links-section {
+  margin-right: 20px;
+  text-align: right;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+  width: 259px;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.lnks {
+  color: #344563;
+}
+
+.main-section {
+  display: flex;
+  align-items: center;
+  justify-content: right;
+  background-image: url(./elements/hdrbckgrnd.svg);
+  background-repeat: no-repeat;
+  width: 1440px;
+  height: 846px;
```

Comment:
- [ ] Please note that your desktop refactoring should not alter the initial mobile version of the application. I Understand you've put in a great deal of work 💯 , However, the design, layout, and functionality of the site or app may need to be optimized for different screen sizes and input methods. The desktop version may be optimized for larger screens and keyboard and mouse input, while the mobile version may be optimized for smaller screens and touch input. Additionally, mobile users may have different browsing habits and needs than desktop users, so a separate mobile version can provide a better user experience for those users.

# Your design 
![image](https://user-images.githubusercontent.com/50990925/213593753-0dd467e6-99ea-4b1d-b39f-147d16f2a49e.png)